03-path-application-development/309-deploying-a-chart-repository/readme.adoc

```diff
@@ -0,0 +1,264 @@
+= Deploying a Chart Repository for Helm Charts
+:toc:
+:icons:
+:linkcss:
+:imagesdir: ../../resources/images
+
+You learned about using `helm` in the module
+link:../../03-path-application-development/306-app-management-with-helm/readme.adoc[306
+Application Management with Helm] this module will teach you how to deploy your
+own Chart Repository to your Kubernetes cluster so that you can deploy custom
+applications.
+
+== Prerequisites
+
+This chapter uses a cluster with 3 master nodes and 5 worker nodes as described here: link:../cluster-install#multi-master-multi-node-multi-az-gossip-based-cluster[multi-master, multi-node gossip based cluster].
+
+All configuration files for this chapter are in the `sample` directory. Make
+sure to `cd` into this directory before running any commands.
+
+This chapter expects that you have gone through 
+link:../../03-path-application-development/306-app-management-with-helm/readme.adoc[306
+Application Management with Helm] meaning you should have installed `helm`.
+
+== ChartMuseum
+
+Chart Museum is an Open Source Chart Repository, it helps you to deploy your
+custom charts to an Amazon S3 bucket and allows you to easily access your charts
+like you would a `stable` release from something like 
+link:https://hub.kubeapps.com/[Kubeapps]. 
+
+== Installing ChartMuseum
+
+To install `chartmuseum` you first need to add the incubator helm repo.
+
+    $ helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com
```

Comment:
Is there no vendor-agnostic URI for this repo?

Reply:
Not that I saw.